src/main/java/POMs/CursMD.java

```diff
@@ -51,4 +57,59 @@ public WebElement getMenuItem(String text) {
         log.info("Searching element with text <" + text + ">");
         return driver.findElement(By.xpath("//ul[contains(@class, 'navbar-nav')]//a[./span[text()='" + text + "']]"));
     }
-}
+
+
+    public WebElement getMenuHeader(String txt) {
+        log.info("Searching element with text <" + txt + ">");
+        return driver.findElement(By.xpath("//div[contains(@class, 'head-menu')]//a[text()='" + txt + "']"));
+    }
+
+    public WebElement getSlogan(String slogan) {
+        log.info("Searching element with text <" + slogan + ">");
+        return driver.findElement(By.xpath("//div[text()='" + slogan + "']"));
+    }
+
+    public WebElement getMainName(String mains) {
+        log.info("Searching element with text <" + mains + ">");
+        return driver.findElement(By.xpath("//*[@id='graficEvolutie']/h1[text() ='" + mains + "']"));
+    }
+
+    public WebElement getOperationCheck(String operations) {
+        log.info("Searching element with text <" + operations + ">");
+        return driver.findElement(By.xpath("//*[@id='content']/div/div[1]/ul//a[text()='" + operations + "']"));
+    }
+
+    public WebElement getConvertorName(String convertor) {
+        log.info("Searching element with text <" + convertor + ">");
+        return driver.findElement(By.xpath("//*[@id='convertor']/h1[text()='" + convertor + "']"));
+    }
+
+    public WebElement getHeaderForm(String headerform) {
+        log.info("Searching element with text <" + headerform + ">");
+        return driver.findElement(By.xpath("//*[@id='bankValute']/h1[text()='" + headerform + "']"));
+    }
+
+    public WebElement getBankFormName(String bankformname) {
+        log.info("Searching element with text <" + bankformname + ">");
+        return driver.findElement(By.xpath("//*[@id='bankBox']/h1[text()='" + bankformname + "']"));
+    }
+
+    public WebElement getExchangeOffices(String exchangeoffices) {
+        log.info("Searching element with text <" + exchangeoffices + ">");
+        return driver.findElement(By.xpath("//*[@id='bankBox']/h1[text()='" + exchangeoffices + "']"));
+    }
+
+    public WebElement getExchangeCurrencies(String currencies) {
+        log.info("Searching element with text <" + currencies + ">");
+        return driver.findElement(By.xpath("//*[@id='content']/div/div[1]/div/h1[text()='" + currencies + "']"));
+    }
+
```

Comment:
These methods could be replaced with only one 
for instance - checkHeader(headerName) and inside findBy.xpath(//h1[text=.....])